services/traction/bdd-tests/features/steps/security.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import json
+
+from behave import *
+from starlette import status
+
+
+@then(
+    '"{tenant_1}" cannot NOT get "{tenant_2}"s contact to "{tenant_2_contact_alias}" by id'
```

Comment:
Should this read "can NOT get" rather than the double negative? It's testing that tenant 1 isn't allowed to see tenant2's contacts right?